src/qililab/chip/chip.py

```diff
@@ -24,7 +27,10 @@
 
 @dataclass
 class Chip:
-    """Chip representation as a graph."""
+    """Chip representation as a graph.
+    This class represents the chip structure of in the runcard and contains all the connectivity information of
+    the chip.
+    """
 
     nodes: list[Node]
```

Comment:
This attribute has no definition in the documentation. Could you include it, please?

Reply:
Addressed.